Resources/doc/first_steps.rst

```diff
@@ -438,7 +438,64 @@ You can use this new method just like the default finder methods of the reposito
     When using a custom repository class, you still have access to the default
     finder methods such as ``find()`` and ``findAll()``.
 
+Service Repositories
+~~~~~~~~~~~~~~~~~~~~
+
+In the previous section, you learnt how to create custom repository classes and how
+to get them using ``DocumentManager``. Another way of obtaining a repository instance
+is to use the repository as a service and inject it as a dependency into other services.
+
+.. code-block:: php
+
+    // src/Acme/StoreBundle/Repository/ProductRepository.php
```

Comment:
Do we want to document the `repositoryClass` argument to the `Document` mapping again to be on the safe side?

Reply:
Not sure, previous chapter is all about that :) On the other hand we could add a comment in the code reminding user to set `repositoryClass`

Reply:
A short comment sounds sensible and should be sufficient if the previous section covers this in more detail 👍

Reply:
Done 👍